adapters/ucfunnel/ucfunnel_test.go

```diff
@@ -0,0 +1,97 @@
+package ucfunnel
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/adapters/adapterstest"
+)
+
+func TestUcfunnelAdapterNames(t *testing.T) {
+	adapter := NewUcfunnelBidder("http://localhost/bid")
+	adapterstest.VerifyStringValue(adapter.Name(), "ucfunnel", t)
+}
+
+func TestSkipNoCookies(t *testing.T) {
+	adapter := NewUcfunnelBidder("http://localhost/bid")
+	status := adapter.SkipNoCookies()
+	if status != false {
+		t.Errorf("actual = %t expected != %t", status, false)
+	}
+}
+
+func TestMakeRequests(t *testing.T) {
```

Comment:
The imp test cases seem indepdent of each other. Please consider breaking up this test up or using table driven tests (https://github.com/golang/go/wiki/TableDrivenTests).